src/Microsoft.Extensions.Options/OptionsServiceCollectionExtensions.cs

```diff
@@ -189,5 +189,32 @@ public static IServiceCollection ConfigureOptions(this IServiceCollection servic
             }
             return services;
         }
+
+        /// <summary>
+        /// Creates an options builder that forwards Configure calls for the same <typeparamref name="TOptions"/> to the underlying service collection.
+        /// </summary>
+        /// <typeparam name="TOptions">The options type to be configured.</typeparam>
+        /// <param name="services">The <see cref="IServiceCollection"/> to add the services to.</param>
+        /// <returns>The <see cref="OptionsBuilder{TOptions}"/> so that configure calls can be chained in it.</returns>
+        public static OptionsBuilder<TOptions> BuildOptions<TOptions>(this IServiceCollection services) where TOptions : class
```

Comment:
I'm not sure about this name.

Reply:
What about these alternatives?
1. `Configure<TOptions>()` (as it would be the only of these overloads that don't take in an extra argument, but it might be confusing that they have different return values)
2. `CreateOptionsBuilder<TOptions>()`
3. `ConfigureOptions<TOptions>()` (although there is already a non-generic overload with this name that takes a type, so it might be very confusing)
4. `BuildOptions<TOptions>()` (just putting it here for contrasting)
5. `ConfiguratorFor<TOptions>()` (probably might make sense to rename `OptionsBuilder` to `OptionsConfigurator` if we choose this extension method name, and maybe even regardless, as this helper doesn't build from scratch, just helps configure).
6. `UseConfigurator<TOptions>()`

Reply:
BTW, reflecting on this, my preference would be option 5.